README.md

```diff
@@ -125,7 +125,9 @@ $ brew install buildkit
 
 To build BuildKit from source, see [`.github/CONTRIBUTING.md`](./.github/CONTRIBUTING.md).
 
-### Starting the `buildkitd` daemon
+For a `buildctl` reference, see [this document](./docs/buildctl.md).
+
+### Starting the `buildkitd` daemon:
```

Comment:
```suggestion
### Starting the `buildkitd` daemon
```

Reply:
Huh, now how did I add that? 🤷‍♂️ 

Fixed.